### PR TITLE
Secrets Sync UI: Small UX copy improvements

### PR DESCRIPTION
**Pull request by hellobontempo**
Small UX changes to clarify copy

### CODE REVIEW

File: ui/lib/kv/addon/components/page/secret/details.hbs [hunk 1]
```diff
@@ -6,10 +6,12 @@
 <KvPageHeader @breadcrumbs={{@breadcrumbs}} @pageTitle={{@path}}>
   <:syncDetails>
     {{#if this.syncStatus}}
-      <Hds::Alert data-test-sync-alert @type="page" @color="neutral" @icon={{false}} as |A|>
+      <Hds::Alert data-test-sync-alert @type="page" @color="neutral" as |A|>
```
Comment (hellobontempo) [inline ui/lib/kv/addon/components/page/secret/details.hbs:9]: Screenshot 2024-01-05 at 12 46 42 PM

**Review by zofskeez — APPROVED** [review]

File: ui/lib/kv/addon/components/page/secret/details.hbs [hunk 2]
```diff
@@ -6,10 +6,12 @@
...
         <A.Title>
-          This secret has been synced from Vault to other destinations, updates to the secret will get automatically synced
-          to destinations.
+          This secret has been synced from Vault to the
+          {{if (eq this.syncStatus.length 1) "destination" "destinations"}}
+          below. Updates to this secret will automatically sync to its
+          {{if (eq this.syncStatus.length 1) "destination" "destinations"}}.
```
[the remark this answers is not in the document]
Comment (hellobontempo) [reply ui/lib/kv/addon/components/page/secret/details.hbs:14]: I'm getting this error for some reason 

```
vendor.js:34570 Uncaught (in promise) Error: Attempted to resolve `pluralize`, which was expected to be a component or helper, but nothing was found.
    at resolveComponentOrHelper (vendor.js:34570:17)
    at encodeOp (vendor.js:36790:18)
    at pushOp (vendor.js:36699:7)
    at vendor.js:36360:9
    at Compilers.compile (vendor.js:34698:7)
    at compileStatements (vendor.js:36703:17)
    at maybeCompile (vendor.js:36678:18)
    at CompilableTemplateImpl.compile (vendor.js:36658:14)
    at VM.compile (vendor.js:44338:50)
    at Object.evaluate (vendor.js:40902:21)

[the remark this answers is not in the document]
Comment (hellobontempo) [reply ui/lib/kv/addon/components/page/secret/details.hbs:14]: Why wouldn't we have access to this helper in the addon engine?

[the remark this answers is not in the document]
Comment (hellobontempo) [reply ui/lib/kv/addon/components/page/secret/details.hbs:14]: I'll try adding `ember-inflector` to the package.json file

[the remark this answers is not in the document]
Comment (hellobontempo) [reply ui/lib/kv/addon/components/page/secret/details.hbs:14]: Yep - that did the trick

[the remark this answers is not in the document]
Comment (zofskeez) [reply ui/lib/kv/addon/components/page/secret/details.hbs:14]: Ah yes my bad I should have mentioned that's where it comes from.

[the remark this answers is not in the document]
Comment (hellobontempo) [reply ui/lib/kv/addon/components/page/secret/details.hbs:14]: It's a little more verbose, but there ya have it! 
```hbs
 {{pluralize this.syncStatus.length "destination" without-count=(eq this.syncStatus.length 1)}}
  below. Updates to this secret will automatically sync to its
 {{pluralize this.syncStatus.length "destination" without-count=true}}.

[the remark this answers is not in the document]
Comment (zofskeez) [reply ui/lib/kv/addon/components/page/secret/details.hbs]: You shouldn't need to use `without-count`. Per [the docs](https://github.com/emberjs/ember-inflector):
```hbs
{{pluralize 1 "taco"}} -> 1 taco
{{pluralize 2 "taco"}} -> 2 tacos
```

**Pull request merged**